src/stanc/stanc.ml

```diff
@@ -79,25 +79,43 @@ let add_file filename =
 (** ad directives from the given file. *)
 let use_file filename =
   let ast =
-    try Parse.parse_file Parser.Incremental.program filename
-    with Errors.SyntaxError err ->
-      Errors.report_syntax_error err ;
+    try 
+      match Parse.parse_file Parser.Incremental.program filename with
+      | Result.Ok ast -> ast
```

Comment:
So does it say OK even if there is a lexing error, for instance?

Reply:
No, lexing errors still raise exceptions at the moment which is outside the context of validation.